### DISCUSSION TITLE
How do you change the top bar color for quarto dashboard?

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-cli#8351] [Q&A] [answered]
### Description

Hi,

I want my top bar to be red

I tried creating 

theme.scss

```scss

/*-- scss:functions --*/

/*-- scss:defaults --*/
$navbar-color:  rgba(216, 6, 33 .65);

/*-- scss:rules --*/

```

my qmd

```

---
title: "title"
format: dashboard
editor: visual
date-modified: today
standalone: true
embed-resources: true
theme: theme-custom.scss
---


# something

a dashboard

```

I don't understand how it works.

Error: $blue: 33 0.65 is not a number.
    ╷
866 │ $navbar-color:  rgba(216, 6, 33 .65);
    │                 ^^^^^^^^^^^^^^^^^^^^
    ╵

I tried following https://quarto.org/docs/output-formats/html-themes-more.html but I do not understand how to make it work.

**Comment by mcanouil** [comment]
You need to use existing variables: https://quarto.org/docs/dashboards/theming.html

**Reply by olivroy** [reply]
Hi, Using

 https://github.com/twbs/bootstrap/blob/af092931f09daadc898b77e78728cd763b974f5d/scss/_variables.scss#L1221

`navbar-color-light` yields the same error message.

```scss

/*-- scss:functions --*/

/*-- scss:defaults --*/
$navbar-color-light:  rgba(216, 6, 33 .65);

/*-- scss:rules --*/
```

**Reply by mcanouil** [reply, marked as answer]
Where do you see `$navbar-color-light` in the link I shared exactly?
Dashboard SCCS variables are listed in the dashboard part of the documentation, here is the link again: <https://quarto.org/docs/dashboards/theming.html#navigation>.

Also your SCSS is not valid, *i.e.*, `33 .65` is not a number, nor two values.

| Syntax 1 | Syntax 2 |
|--------|--------|
| <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/4767f1d7-09a4-4c0a-93fc-1c5180363a55"> | <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5ffd434f-f416-4030-8c39-6565f9d4b123"> |

**Reply by olivroy** [reply]
Sorry, thanks for your help. I didn't look at the right place. I opened https://github.com/quarto-dev/quarto-web/pull/984.

It is just that when I saw the [example code block](https://quarto.org/docs/dashboards/theming.html#theme-options), the only thing I saw was the link to https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss which was misleading
